.github/workflows/test.yml

```
@@ -22,7 +22,7 @@ jobs:
 
       - uses: subosito/flutter-action@v2
         with:
-          channel: stable
+          channel: ${{vars.FLUTTER_CHANNEL}}
```

Please add `flutter-version: ${{ vars.FLUTTER_VERSION }}` here too
```suggestion
          channel: ${{vars.FLUTTER_CHANNEL}}
          flutter-version: ${{ vars.FLUTTER_VERSION }}
```